internal/services/v1/relationships.go

```diff
@@ -28,14 +28,27 @@ import (
 	"github.com/authzed/spicedb/pkg/zedtoken"
 )
 
+// PermissionsServerConfig is configuration for the permissions server.
+type PermissionsServerConfig struct {
+	// MaxUpdatesPerWrite holds the maximum number of updates allowed per
+	// WriteRelationships call.
+	MaxUpdatesPerWrite uint16
+
+	// MaxPreconditionsCount holds the maximum number of preconditions allowed
+	// on a WriteRelationships or DeleteRelationships call.
+	MaxPreconditionsCount uint16
+}
+
 // NewPermissionsServer creates a PermissionsServiceServer instance.
 func NewPermissionsServer(
 	dispatch dispatch.Dispatcher,
 	defaultDepth uint32,
+	config PermissionsServerConfig,
```

Comment:
Looking at the existing code, the servers all just take in a list of their "config" so that the Server object itself holds all the options.

Why not do that here? Or if we're going have config objects, why not put all of the config in them?

Reply:
I'm fine with moving the defaultDepth and other non-dispatch values into the config, if you prefer